Resources/TuxHook2/inject.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```

Comment:
`#!/usr/bin/env sh`, from what I see you don't use any bash extensions. POSIX compatibility ftw!

I'd also append `set -eu`.